src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,159 @@
+    Distractor1: "It provides a platform for game development",
+    Distractor2: "It offers free stock images",
+    Distractor3: "It is a social media network",
+    Explanation: 
+    "The 'a11yproject' is a valuable resource for web developers looking to enhance the accessibility of their projects.",
+    Link: "https://www.freecodecamp.org/news/accessibility-testing-tools-for-ada-compliance/"
+  },
+  {
+    Question: "Which tool allows you to check foreground and background color values for AA and AAA color contrast ratios for small and large text?",
+    Answer: 
+    "WebAIM Color Contrast Checker",
+    Distractor1: "Accessible Color Palette Builder",
+    Distractor2: "Randoma11y Color Generator",
+    Distractor3: "Check My Colours",
+    Explanation: 
+    "The WebAIM Color Contrast Checker is a tool to verify color contrast, a crucial aspect of web accessibility.",
+    Link: "https://www.freecodecamp.org/news/accessibility-testing-tools-for-ada-compliance/"
+  },
+  {
+    Question: "Why is web accessibility important for SEO (Search Engine Optimization)?",
+    Answer: 
+      "It helps search engines understand and index website content better.",
+    Distractor1: "It decreases website load times.",
+    Distractor2: "It automatically improves website design.",
+    Distractor3: "It increases advertising revenue.",
+    Explanation: 
+      "Accessibility practices that make content understandable to screen readers also benefit search engines in understanding and indexing website content.",
+    Link: "https://www.freecodecamp.org/news/common-accessibility-errors-and-how-to-fix-them/"
+  },
+  {
+    Question: "Which accessibility problem is associated with empty buttons, particularly those without text for screen reader users?",
+    Answer: 
+      "Empty buttons",
+    Distractor1: "Low color contrast",
+    Distractor2: "Missing alternative text for images",
+    Distractor3: "Missing form input labels",
+    Explanation: 
+      "Empty buttons that lack text for screen reader users are an accessibility issue.",
+    Link: "https://www.freecodecamp.org/news/common-accessibility-errors-and-how-to-fix-them/"
+  },
+  {
+    Question: "What is the primary reason for adding a language attribute to the HTML tag of a webpage?",
+    Answer: 
+      "Screen readers use the document language to pronounce words, and it's good for SEO.",
+    Distractor1: "Language attributes improve website aesthetics.",
+    Distractor2: "Language attributes speed up website loading.",
+    Distractor3: "Language attributes provide translations for the page.",
+    Explanation:
+      "Adding a language attribute to the HTML tag helps screen readers pronounce words correctly and benefits SEO.",
+    Link: "https://www.freecodecamp.org/news/common-accessibility-errors-and-how-to-fix-them/"
+  },
```

Comment:
we already have a similar question in the file.
please remove this

Reply:
sure